src/components/GlobalPreferences/Notifications/NotificationsLogin.js

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { Box, Button, GU, TextInput, Text, Info, IconMail } from '@aragon/ui'
+import { login } from './notification-service-api'
+import { AUTH_PREVERIFY } from './constants'
+import { getEthNetworkType } from '../../../local-settings'
+import notificationSvg from './notifications.svg'
+import checkmarkSvg from './check-mark.svg'
+
+export default function NotificationsLogin({
+  dao,
+  authState,
+  email,
+  onEmailChange,
+}) {
+  const [inputEmail, setInputEmail] = useState('')
+  const [apiError, setApiError] = useState(null)
+  const network = getEthNetworkType()
+
+  const handleEmailChange = e => {
+    setInputEmail(e.target.value)
+  }
+
+  const handleLogin = async e => {
+    try {
+      await login({ email: inputEmail, dao, network })
+      onEmailChange(inputEmail)
+    } catch (e) {
+      setApiError(e.message)
+      console.error('Failed to login', e)
+    }
+  }
+
+  if (authState === AUTH_PREVERIFY) {
+    return (
+      <Box heading="Email notifications">
+        <NotificationImage />
+        <div>
+          <Checkmark />
+          <Text>Awaiting verification. Please check your email!</Text>
+          <br />
+          <Text size="xsmall">
+            We’ve sent an email to {email}. Verify your email address so you can
+            manage your notifications subscriptions.
+          </Text>
+        </div>
+      </Box>
+    )
+  }
+  return (
+    <Box heading="Email notifications">
+      <NotificationImage />
+      {apiError && <Info mode="error">Error logging in:{apiError}</Info>}
+      <Label>Email address</Label>
+      <TextInput
+        css={`
+          width: 80%;
+          margin-right: 9px;
+          margin-bottom: 10px;
+        `}
+        type="email"
+        placeholder="you@example.com"
+        wide
+        value={inputEmail}
+        onChange={handleEmailChange}
+      />
+      <Button onClick={handleLogin}>
```

Comment:
We probably need a container to set vertical alignment here:
![image](https://user-images.githubusercontent.com/3072458/62776412-d9099000-baaa-11e9-8fe2-004466d6739e.png)